packages/types/src/transaction.ts

```diff
@@ -132,4 +132,10 @@ export interface TransactionMetadata {
     sentry?: string;
     thirdparty?: string;
   };
+
+  /** For transactions tracing server-side request handling, the request being tracked. */
+  request?: {
+    [key: string]: any;
```

Comment:
Same as prev comment, what about going with a type/interface?

Reply:
I actually think I'm going to simplify matters and just store the URL, rather than the entire request, because that's all I really need.